### DISCUSSION TITLE
Blocked domain

### DISCUSSION THREAD

**Discussion by katastrophe92** [anacrolix/dht#71] [Q&A]
Hello 👋. NextDNS is blocking `dht.aelitis.com`. I am still able to download though. Should I place it on the DNS allowlist? Hope someone can help me. Thanks!

**Comment by anacrolix** [comment]
No, there are 5 or 6 other global bootstrap addrs that should be adequate. If a significant number of those are missing (particularly in the address family you need like ipv4/6), then I would start looking into whitelisting.
